Applications/CLI/ogs.cpp

```diff
@@ -109,7 +113,11 @@ int main(int argc, char *argv[])
         {
             ApplicationsLib::LinearSolverLibrarySetup
                 linear_solver_library_setup(argc, argv);
-
+#if defined(USE_PETSC)
+            vtkMPIController* controller = vtkMPIController::New();
```

Comment:
I have tried to put this vtkMPIController stuff as a member of `` linear_solver_library_setup``,  it did not work. The place here for vtkMPIController stuff  makes correct pvtu  output.